layers/+os/osx/keybindings.el

```diff
@@ -41,24 +41,30 @@
 
   ;; Set internal variables according to the given config variables
   (cl-loop for (key-var . internal-var) in modifier-keys do
-           (let ((key-value (symbol-value key-var)))
-             (when (member key-value allowed-values)
-               (setf (symbol-value internal-var) key-value))))
+    (let ((key-value (symbol-value key-var)))
+      (when (member key-value allowed-values)
+        (setf (symbol-value internal-var) key-value))))
 
   (defun kbd-mac-command (keys)
     "Wraps `kbd' function with Mac OSX compatible Command-key (⌘).
+     (when osx-swap-option-and-command
+        (cl-rotatef mac-command-modifier mac-option-modifier))
+
 KEYS should be a string such as \"f\" which will be turned into values
 such as \"H-f\", \"s-f\", or \"A-f\" depending on the value of
 `mac-commmand-modifier' which could be `hyper', `super', or `alt'.
 KEYS with a string of \"C-f\" are also valid and will be turned into
 values such as \"H-C-f\".
 Returns nil if `mac-command-modifier' is set to `none' or something
 other than the three sane values listed above."
+
```

Comment:
Remove this empty line, there shouldn't be an empty line between a functions dosctring and the function body.
```suggestion
```